includes/class-wcs-att-cart.php

```diff
@@ -129,6 +132,11 @@ public static function get_cart_subscription_schemes() {
 			if ( WCS_ATT_Product::is_subscription_product_type( $cart_item[ 'data' ] ) ) {
 				return false;
 			}
+
+			// Renewing/Resubscribing?
+			if ( isset( $cart_item[ 'subscription_renewal' ] ) || isset( $cart_item[ 'subscription_initial_payment' ] ) || isset( $cart_item[ 'subscription_resubscribe' ] ) ) {
+				return false;
```

Comment:
Not 100% sure about this.

I think I will need to test how cart subscriptions behave when re-subscribing -- with this in place I have the impression that cart-level schemes will not be appended to product-level schemes on cart item objects.

The reason I added this was that I wanted to prevent cart subscription options from showing up, but in hindsight this is probably a bad place to deal with this.